lib/mindee/parsing/prediction/financial_document/financial_document_v1.rb

```diff
@@ -0,0 +1,95 @@
+# frozen_string_literal: true
+
+require_relative '../invoice/invoice_v4'
+
+module Mindee
+  module Prediction
+    # Financial document.
+    class FinancialDocumentV1 < InvoiceV4
```

Comment:
I've gone with the approach of inheriting from Invoicev4, but maybe it's better to duplicate the code and extract common parts? I'm OK with both

Reply:
Would be better to duplicate the code, as there could be independent updates to each API.

We don't create a new class unless there are breaking changes, so for example adding some fields in a V1.1 (API version), we would just add the fields to the current class.

Reply:
I'll make the changes then